src/libraries/System.Private.CoreLib/gen/ProductVersionInfoGenerator.cs

```diff
@@ -12,18 +12,25 @@ public partial class ProductVersionInfoGenerator : IIncrementalGenerator
     {
         public void Initialize(IncrementalGeneratorInitializationContext context)
         {
-            context.RegisterPostInitializationOutput(ctx =>
-            {
-                string? informationalVersion = typeof(ProductVersionInfoGenerator).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion;
+            var informationalVersionProvider = context.AnalyzerConfigOptionsProvider.Select((options, _) =>
```

Comment:
```suggestion
            string informationalVersionProvider = context.AnalyzerConfigOptionsProvider.Select((options, _) =>
```
Nit: Repo coding conventions

Reply:
This is not a string but Roslyn's `IncrementalValueProvider<T>`: https://github.com/dotnet/roslyn/blob/main/docs%2Ffeatures%2Fincremental-generators.md#incrementalvaluesprovidert

Reply:
ah sorry, I missed that this was about the coding conventions: https://github.com/dotnet/runtime/pull/110997